tests/llmcompressor/transformers/sparsification/test_compress_tensor_utils.py

```diff
@@ -171,34 +179,41 @@ def test_quant_model_reload(format, dtype, tmp_path):
         oneshot_device=device,
         precision=dtype,
     )
-
-    model = SparseAutoModelForCausalLM.from_pretrained(
-        tmp_path / "oneshot_out", torch_dtype=dtype
+    save_path = tmp_path / "oneshot_out"
+    model.save_pretrained(
+        save_path,
+        quantization_format=format,
+        save_compressed=True,
     )
 
-    for _, module in model.named_modules():
+    loaded_model = AutoModelForCausalLM.from_pretrained(
+        save_path,
+        torch_dtype=dtype,
+    )
+    for _, module in loaded_model.named_modules():
         if hasattr(module, "quantization_scheme"):
             assert module.weight.dtype == dtype
             assert module.quantization_status == QuantizationStatus.FROZEN
 
+    config = AutoConfig.from_pretrained(save_path)
+    compression_config = getattr(config, QUANTIZATION_CONFIG_NAME, None)
+    quant_config = ModelCompressor.parse_quantization_config(compression_config)
+    assert quant_config["format"] == format
+
     model.save_pretrained(
         tmp_path / "compress_out",
         quantization_format=format,
-        save_compressed=True,
+        save_compressed=False,
     )
 
-    config = AutoConfig.from_pretrained(tmp_path / "compress_out")
-    compression_config = getattr(config, QUANTIZATION_CONFIG_NAME, None)
-    quant_config = ModelCompressor.parse_quantization_config(compression_config)
-    assert quant_config["format"] == format
-
-    dense_model = SparseAutoModelForCausalLM.from_pretrained(
```

Comment:
Why are we removing valid test cases?

Reply:
This is the change to AutoModelForCuasalLM